src/frontends/onnx/frontend/src/op/blackmanwindow.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "onnx_import/core/node.hpp"
+
+OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
Could you align this part with other headers, like  https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/asinh.hpp

```
#pragma once

#include "openvino/core/deprecated.hpp"
OPENVINO_SUPPRESS_DEPRECATED_START
```
for all headers